src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,51 @@
+body {
+  margin: 0;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: calc(7 * $day-size + 6 * $gap + 2 * $padding);
+  padding: 5px;
+  padding: $padding;
+  margin: auto;
+  gap: $gap;
+
+  @each $day, $index in $days {
```

Comment:
The loop `@each $day, $index in $days` assumes that `$days` is a list or map defined elsewhere. Make sure `$days` is defined to avoid errors during compilation.